### DISCUSSION TITLE
Release notes for pre-release 3.4

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#2401] [Documentation]
Here are changes in pre-release 3.4 builds (https://www.sumatrapdfreader.org/prerelease):

* Command Palette with `Ctrl - K`
* configurable keyboard shortcuts https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts
* better support for epub files using mupdf's epub engine. Adds text selection and search in ebook files
* `-all-users` cmd-line arg for installer for system-wide install
* added ability to search / translate selected text with web services
  we have few built-in and you can add your own via
  https://www.sumatrapdfreader.org/settings/settings3-4#SelectionHandlers
* added `Annotations.TextIconColor` and `TextIconType` advanced settings
* added `Annotations.UnderlineColor` advanced setting
* added `Annotations.DefaultAuthor` advanced setting
  https://github.com/sumatrapdfreader/sumatrapdf/issues/2025
* `Shift + i` keyboard shortcuts inverts document colors
* `u` and `Shift + u` keyboard shortcuts adds underline annotation for currently selected text
  https://github.com/sumatrapdfreader/sumatrapdf/issues/2157
* `Delete` / `Backspace` keyboard shortcuts delete an annotation under mouse cursor
* respect zoom level in PDF destination links 3aa18aaf89caef7dd28529682d9344ef89422100 
* support .svg files
* faster scrolling with mouse wheel when cursor over scrollbar
* add `-search` cmd-line option and `[Search("<file>", "<search-term>")]` DDE command
* when showing document properties press `Ctrl-d` will show font information for PDF files
* support opening .heic image files (if Windows has heic codec installed)
* add experimental smooth scrolling (enabled with SmoothScrolling advanced setting)

**Comment by medongaw** [comment]
hi kjk, thanks so much for making sumatrapdf to begin with.
I've just installed this prerel build to test if it's any better than 3.3 stable build
and it was. not sure it is just me but epub seems to open much faster, even the layout looks more like pdf file.
I want to keep using this but want to make one request to make this even better.

can I change the font and its size for epub files? 
in the previous build I could open settings file and change those values to my liking
and now I found this 

	EbookFontName = default
	EbookFontSize = 10

I changed it to like Georgia and 13 but nothing's changed.
I assume FixedPageUI only defines pdf and other formats but not epub.

but It does not even change pdf font and size(I assume pdf file is born to work that way unlike epub).

any solutions would be greatly appreciated!

**Reply by kjk** [reply]
those are not yet implemented in new ebook engine
